src/components/Article/Article.js

```diff
@@ -14,11 +13,11 @@ class Article extends Component {
                 title="automatic wi-fi security"
                 />
                 <ArticleText
-                text="mymymymmymymymyymymymmymymymyymymymymyymymymyymymymyymymymymymymm"
+                text="test"
                 />
                 <Button title="start now" status={true}/>
                 <Button title="request demo" status={true}/>
```

Comment:
Create better name for button component, i dont see the differences between regular button.